### DISCUSSION TITLE
Camel-case vs snake case

### DISCUSSION THREAD

**Discussion by prasannakumar1989** [go-webauthn/webauthn#155] [Q&A] [answered]
Are Camel case JSON Keys mandated by the spec? 
https://github.com/go-webauthn/webauthn/blob/9cb3504fb344bae988f6c073b054ed3b2b6d99f9/protocol/options.go#L47

**Comment by james-d-elliott** [comment, marked as answer]
My recollection is that it's never expressly mentioned. However the case we use is the same case that's used in the WDL's as far as I am aware, and it would make sense that not using the same case as specified in the WDL's would likely fail in most browsers as they are probably only expecting the format specified by the WDL's.
